[2025-10-31T22:35:06.829+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-31T22:35:06.848+0700] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: absa_streaming_lifecycle_full.deploy_producer manual__2025-10-31T15:35:04.749162+00:00 [queued]>
[2025-10-31T22:35:06.853+0700] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: absa_streaming_lifecycle_full.deploy_producer manual__2025-10-31T15:35:04.749162+00:00 [queued]>
[2025-10-31T22:35:06.854+0700] {taskinstance.py:2303} INFO - Starting attempt 1 of 3
[2025-10-31T22:35:06.861+0700] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): deploy_producer> on 2025-10-31 15:35:04.749162+00:00
[2025-10-31T22:35:06.868+0700] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1004) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-31T22:35:06.869+0700] {standard_task_runner.py:63} INFO - Started process 1014 to run task
[2025-10-31T22:35:06.874+0700] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'absa_streaming_lifecycle_full', 'deploy_producer', 'manual__2025-10-31T15:35:04.749162+00:00', '--job-id', '293', '--raw', '--subdir', 'DAGS_FOLDER/absa_streaming_lifecycle_full.py', '--cfg-path', '/tmp/tmpdt86w39m']
[2025-10-31T22:35:06.878+0700] {standard_task_runner.py:91} INFO - Job 293: Subtask deploy_producer
[2025-10-31T22:35:06.938+0700] {task_command.py:426} INFO - Running <TaskInstance: absa_streaming_lifecycle_full.deploy_producer manual__2025-10-31T15:35:04.749162+00:00 [running]> on host f7bb1ad77e89
[2025-10-31T22:35:07.041+0700] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='absa_streaming_lifecycle_full' AIRFLOW_CTX_TASK_ID='deploy_producer' AIRFLOW_CTX_EXECUTION_DATE='2025-10-31T15:35:04.749162+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-31T15:35:04.749162+00:00'
[2025-10-31T22:35:07.043+0700] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-31T22:35:07.044+0700] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-10-31T22:35:07.045+0700] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'bash -c "timeout 45m /opt/***/projects/absa_streaming/scripts/run_producer.sh"']
[2025-10-31T22:35:07.065+0700] {subprocess.py:86} INFO - Output:
[2025-10-31T22:35:07.074+0700] {subprocess.py:93} INFO - timeout: failed to run command ‘/opt/***/projects/absa_streaming/scripts/run_producer.sh’: Permission denied
[2025-10-31T22:35:07.075+0700] {subprocess.py:97} INFO - Command exited with return code 126
[2025-10-31T22:35:07.080+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-31T22:35:07.099+0700] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 460, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 126.
[2025-10-31T22:35:07.109+0700] {taskinstance.py:1205} INFO - Marking task as UP_FOR_RETRY. dag_id=absa_streaming_lifecycle_full, task_id=deploy_producer, execution_date=20251031T153504, start_date=20251031T153506, end_date=20251031T153507
[2025-10-31T22:35:07.121+0700] {standard_task_runner.py:110} ERROR - Failed to execute job 293 for task deploy_producer (Bash command failed. The command returned a non-zero exit code 126.; 1014)
[2025-10-31T22:35:07.164+0700] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-10-31T22:35:07.188+0700] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-31T22:35:07.189+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
